Examples/Algorithms/Fitting/src/FittingAlgorithm.cpp

```diff
@@ -71,7 +71,8 @@ ActsExamples::ProcessCode ActsExamples::FittingAlgorithm::execute(
     const auto& protoTrack = protoTracks[itrack];
     const auto& initialParams = initialParameters[itrack];
 
-    // We can have empty tracks which must give empty fit results
+    // We can have empty tracks which must give empty fit results so the number
+    // of entries in input and output containers matches.
```

Comment:
Redundant.

Reply:
Going through this code, it was not immediately clear why this has to be the case. I needed to check the full loop to understand this. Commented is updated to make it clearer for future readers.